frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.unit.spec.js

```diff
@@ -11,15 +11,62 @@ registerVisualizations();
 const setup = (series, width) =>
   renderWithProviders(<Visualization rawSeries={series} width={width} />);
 
-const series = ({ rows, insights }) => {
+const series = ({ rows, insights, field }) => {
   const cols = [
     DateTimeColumn({ name: "Month" }),
     NumberColumn({ name: "Count" }),
+    NumberColumn({ name: "Sum" }),
+  ];
+  return [
+    {
+      card: {
+        display: "smartscalar",
+        visualization_settings: { "scalar.field": field },
```

Comment:
Could you please add a test case to verify existing charts that do not have `scalar.field` setting works correctly?

Reply:
3d259d4850ebf6f01bb46744a2bc621ab0dac33c